MapboxAndroidDemo/src/china/java/com/mapbox/mapboxandroiddemo/examples/SimpleChinaMapViewActivity.java

```diff
@@ -55,17 +66,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
     switch (item.getItemId()) {
       case R.id.menu_streets:
         if (mapboxMap != null) {
-          mapboxMap.setStyleUrl(ChinaStyle.MAPBOX_STREETS_CHINESE);
+          mapboxMap.setStyle(new Style.Builder().fromUrl(ChinaStyle.MAPBOX_STREETS_CHINESE));
         }
         return true;
       case R.id.menu_dark:
         if (mapboxMap != null) {
-          mapboxMap.setStyleUrl(ChinaStyle.MAPBOX_DARK_CHINESE);
+          mapboxMap.setStyle(new Style.Builder().fromUrl(ChinaStyle.MAPBOX_DARK_CHINESE));
         }
         return true;
       case R.id.menu_light:
         if (mapboxMap != null) {
-          mapboxMap.setStyleUrl(ChinaStyle.MAPBOX_LIGHT_CHINESE);
+          mapboxMap.setStyle(new Style.Builder().fromUrl(ChinaStyle.MAPBOX_LIGHT_CHINESE));
```

Comment:
for 3 occurences above maybe just use `mapboxMap#setStyle(String styleUrl)` instead?

Reply:
None of the China style URLs are part of the list of URLs which `setStyle()` accepts. Have to use the builder's `fromUrl()` for now.

![Uploading Screen Shot 2019-01-25 at 10.29.52 AM.png…]()